src/authorization/graphql/user.graphql

```diff
@@ -47,8 +47,12 @@ permissions: [String]!,
 operation: OperationType
 }
 
+input FilterUserInput {
+  searchTerm: String
+}
+
 type Query {
-  getUsers: [User]!
+  getUsers(input: FilterUserInput): [User]!
```

Comment:
Can we accept a generic SearchTerm here?

Example:
```
getUsers(input: {
    "and": [
        "email": {
            "contains" :"search by search term that contains"
        },
        "firstName": {
            "ne": "not equal to this search term"
        }
    ]
}) {
    id
}
```
Should return users which satisfy both "email contains" and "first name not equal to"